src/VoxaEvent.ts

```diff
@@ -87,6 +90,44 @@ export abstract class VoxaEvent implements IVoxaEvent {
     this.initLogger(logOptions);
   }
 
+  public async getUserInformation(): Promise<any> {
+    if (_.has(this, "google")) {
+      return this.getUserInformationWithGoogle();
+    }
+
+    return await this.getUserInformationWithLWA();
+  }
+
+  public getUserInformationWithGoogle(): any {
+    const result: any = jwt.decode(this.user.idToken);
```

Comment:
`this.user.idToken` ??

Reply:
It's the variable that has the JSON Web Token with the user's Google information. It's not an accessToken

Reply:
yes but, from the dialogflow implementation for the user object, seems like this is not correct https://github.com/VoxaAI/voxa/blob/v3/src/platforms/dialogflow/DialogFlowEvent.ts#L91

perhaps you're looking for `voxaEvent.dialogflow.conv.user.idToken` ?